scene/gui/file_dialog.cpp

```diff
@@ -1490,6 +1612,8 @@ FileDialog::FileDialog() {
 	tree->connect("item_activated", callable_mp(this, &FileDialog::_tree_item_activated));
 	tree->connect("nothing_selected", callable_mp(this, &FileDialog::deselect_all));
 	dir->connect("text_submitted", callable_mp(this, &FileDialog::_dir_submitted));
+	filename_filter->connect("text_changed", callable_mp(this, &FileDialog::_filename_filter_changed).unbind(1));
```

Comment:
I get this error when I try to build with your changes
```editor/gui/editor_file_dialog.cpp:2417:50: error: 'class SceneStringNames' has no member named 'text_changed'; did you mean 'texture_changed'?
 2417 |         filename_filter->connect(SceneStringName(text_changed), callable_mp(this, &EditorFileDialog::_filename_filter_changed).unbind(1));
      |                                                  ^~~~~~~~~~~~```

Reply:
You need to rebase, your branch is months behind

Reply:
You should rebase

Reply:
Yes, good to do with some regularity if code changes that might affect your code, as you are dealing with pretty widely used things it could have changed even if there are no conflicts, conflicts aren't the only time you need to rebase